### PR TITLE
fix: fixed unit test cases

### PR DESCRIPTION
**Pull request by omkarK06**
## Release Notes

- **New Features**
  - Enhanced `SessionViewer` component for improved readability in method calls.
  - Additional fields in stream settings for better configuration management.

- **Bug Fixes**
  - Adjusted expected outputs in test cases for `LogStream` and `Schema` components.

- **Refactor**
  - Updated dependencies for testing tools, transitioning from Jest to Vitest.
  - Improved structure and presentation in the `schema.vue` component.

- **Chores**
  - Modified testing configuration paths and updated inline dependencies.

### CODE REVIEW

File: web/src/views/RUM/SessionViewer.vue [hunk 1]
```diff
@@ -284,11 +290,14 @@ const getSessionEvents = () => {
...
-      page_type: "logs",
-    }, "RUM")
+    .search(
+      {
+        org_identifier: store.state.selectedOrganization.identifier,
+        query: req,
+        page_type: "logs",
+      },
+      "RUM",
+    )
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/src/views/RUM/SessionViewer.vue]: Fixed

File: web/src/test/unit/components/logstream/Schema.spec.ts [hunk 2]
```diff
@@ -144,10 +151,10 @@ describe("Streams", async () => {
       .find("tbody")
       .find('[data-test="schema-stream-log-field-fts-key-checkbox"]');
     expect(
-      logCheckbox.find(".q-checkbox__inner--truthy").exists()
+      logCheckbox.find(".q-checkbox__inner--truthy").exists(),
     ).toBeTruthy();
     expect(
-      messageCheckbox.find(".q-checkbox__inner--truthy").exists()
+      messageCheckbox.find(".q-checkbox__inner--truthy").exists(),
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/src/test/unit/components/logstream/Schema.spec.ts:157]: Added todo

File: web/src/test/unit/components/logstream/Schema.spec.ts [hunk 3]
```diff
@@ -49,6 +49,7 @@ describe("Streams", async () => {
 
   it("Should display stream title", () => {
     const streamTitle = wrapper.find('[data-test="schema-stream-title-text"]');
+    console.log(streamTitle.text());
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/src/test/unit/components/logstream/Schema.spec.ts:52]: Fixed

File: web/vite.config.ts [hunk 4]
```diff
@@ -176,9 +176,9 @@ export default defineConfig({
   test: {
     enable: true,
     global: true,
-    setupFiles: "src/test/unit/helpers/setupTests.ts",
+    setupFiles: "test/unit/helpers/setupTests.ts",
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/vite.config.ts:179]: Works fine

**Review by nikhilsaikethe — APPROVED** [review]

**Pull request merged**